debezium-connector-db2/src/main/java/io/debezium/connector/db2/Db2ObjectNameQuoter.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.db2;
+
+/**
+ * By default, names in Db2 are upper case. Unless quoted, 
+ * any name in a SQL statement will interpreted as the equivalent upper case string.
+ * 
+ * This funations quotes a table or schema name in Db2 if the name contains 
```

Comment:
```suggestion
 * This function quotes a table or schema name in Db2 if the name contains 
```

(though the comment would be better located at the method itself?)

Reply:
Good catch 👍